src/edi_energy_scraper/__init__.py

```diff
@@ -106,33 +106,35 @@ async def _download_and_save_pdf(self, file_basename: str, link: str) -> Path:
                     raise
                 await asyncio.sleep(delay=randint(5, 10))  # cool down...
 
-        file_path = self._get_file_path(file_name=file_name)
-        Path.mkdir(file_path.parent, parents=True, exist_ok=True)
-
-        # Save file if it does not exist yet
-        if not os.path.isfile(file_path):
-            with open(file_path, "wb+") as outfile:  # pdfs are written as binaries
-                _logger.debug("Saving new PDF %s", file_path)
-                outfile.write(response_content)
-            return file_path
-
-        # First fix, different file types do just the same as before, only with correct file extension
-        if not file_name.endswith(".pdf"):
-            with open(file_path, "wb+") as outfile:
-                _logger.debug("Saving %s", file_path)
-                outfile.write(response_content)
-            return file_path
-
-        # Check if metadata has changed
-        metadata_has_changed = self._have_different_metadata(response_content, file_path)
-        if metadata_has_changed:  # delete old file and replace with new one
-            _logger.debug("Metadata for PDF %s changed; Replacing it", file_path)
-            os.remove(file_path)
-            with open(file_path, "wb+") as outfile:  # pdfs are written as binaries
-                outfile.write(response_content)
-        else:
-            _logger.debug("Meta data haven't changed for %s", file_path)
-        return file_path
+        # file_path = self._get_file_path(file_name=file_name)
```

Comment:
Kann weg?
```suggestion
```

Reply:
yes das kann weg. Fixed in e7c5d95b04b794320c68cffeb3a1f49a05f6c494